readme.txt

```diff
@@ -61,6 +61,9 @@ Please make sure that your login information is correct, and that you have at le
 
 == Changelog ==
 
+= 7.0 =
+* Force upgrade prompt for users on 6.25.
```

Comment:
This will likely be read by users, can you flesh this out a bit more? Explain _why_ we need to force the upgrade?

Reply:
@bartboy011 Good point, how's that look now?

Reply:
Works for me, thanks!